src/semver/__init__.py

```diff
@@ -4,36 +4,12 @@
 A Python module for semantic versioning. Simplifies comparing versions.
 """
 
-from ._deprecated import (
```

Comment:
Why is the source code changed?

I'm not entirely sure why the change of `pyproject.toml` triggers a reformatting. It shouldn't. The source code is perfectly formatted and the black config wasn't changed. This PR is all about switching from setuptools to hatch(ling) and we should keep it as such.

Could you please check and revert these changes in the source code?

Reply:
I accidentally ran the `style:fmt` script, which runs [isort](https://pycqa.github.io/isort/), and it reformatted (and sorted) the import statements. Using isort would be another suggestion, but again, out of scope of this PR. I'll revert these.

Reply:
Again, no problem.
Actually, I was thinking about isort too. But as you said, we can do that in another PR. :+1: